tck/src/main/java/org/eclipse/microprofile/config/tck/CDIPlainInjectionTest.java

```diff
@@ -51,11 +51,15 @@
  */
 public class CDIPlainInjectionTest extends Arquillian {
 
+    private static final String DEFAULT_PROPERTY_BEAN_KEY = "org.eclipse.microprofile.config.tck.CDIPlainInjectionTest.defaultPropertyBean.configProperty";
```

Comment:
Shouldn't the D in `defaultPropertyBean` be a capital D? It should be the name of the class, right?